.github/workflows/coding-standards.yml

```diff
@@ -6,11 +6,15 @@ on:
       - "v*.*"
       - "master"
       - "feature/*"
+    paths-ignore:
+      - "docs/**"
```

Comment:
I decided to only skip builds if we're only changing documentation: there are too many variables that affect the "should this require a test run" to be feasible (e.g. changes in orchestration, tests, config files, sources, tests, etc.). Listing all the directories in `paths` doesn't seem sensible, and skipping builds for pull requests/commits that ONLY change documentation should be sufficient for the time being.